### ISSUE TITLE
make the tweetfeed the same height as the news?

### ISSUE THREAD

**Issue by Antonialock** [pombase/website#1024]
Should we make the tweetfeed the same height as the news?

Currently you can do a bit of scrolling in the tweets, but that doesn't make sense at the moment since the news section is longer?

<img width="341" alt="screen shot 2018-11-14 at 15 34 17" src="https://user-images.githubusercontent.com/12727506/48492948-cf584200-e822-11e8-8cb8-889906e75e08.png">

**Comment by ValWood** [comment]
Yes, but I think the problem is that  the twitter feed  is a fixed length and news feed is of variable length depending how long the posts are/how many posts we show.
So it might not be possible/easy...

**Comment by kimrutherford** [comment]
I'll work on balancing the heights so the news and twitter feeds match.  I'm not sure how to do it.

For now I've increased the height of the twitter feed.

We can probably move a few items of the news feed on the front page.  It's getting long.

**Comment by mah11** [comment]
I do. not. care. about the relative heights of those two items, but ...

> We can probably move a few items of the news feed on the front page. It's getting long.

I've removed the frontpage flag from news items dated earlier than last month because getting rid of stale news is good in its own right.

**Comment by kimrutherford** [comment]
I can't work out how to do it.  If I ever do, I'll fix it.
